src/App.tsx

```diff
@@ -1,167 +1,120 @@
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+import {
+  Navigate,
+  NavLink,
+  Outlet,
+  Route,
+  Routes,
+  useLocation,
+} from 'react-router-dom';
+import { getPeople } from './api';
+import { PeopleTable } from './components/PeopleTable';
+import { useEffect, useState } from 'react';
+import { Person } from './types';
+import classNames from 'classnames';
+
+const STATUS = {
+  resolved: 'resolved',
+  rejected: 'rejected',
+  idle: 'idle',
+  pending: 'pending',
+} as const;
+
+type Status = (typeof STATUS)[keyof typeof STATUS];
+
+export const App = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [status, setStatus] = useState<Status>('idle');
+  const { pathname } = useLocation();
+  const loadPeople = () => {
+    return getPeople()
+      .then(data => {
+        setPeople(data);
+        setStatus('resolved');
+      })
+      .catch(() => setStatus('rejected'));
+  };
+
+  useEffect(() => {
+    setStatus('pending');
+    loadPeople();
+  }, []);
+
+  return (
+    <>
+      <div data-cy="app">
+        <nav
+          data-cy="nav"
+          className="navbar is-fixed-top has-shadow"
+          role="navigation"
+          aria-label="main navigation"
+        >
+          <div className="container">
+            <div className="navbar-brand">
+              <NavLink
+                className={({ isActive }) =>
+                  classNames('navbar-item', {
+                    'has-background-grey-lighter': isActive,
+                  })
+                }
+                to="/"
+              >
+                Home
+              </NavLink>
+
+              <NavLink
+                className={({ isActive }) =>
+                  classNames('navbar-item', {
+                    'has-background-grey-lighter': isActive,
+                  })
+                }
+                to="people"
+              >
+                People
+              </NavLink>
+            </div>
+          </div>
+        </nav>
+
+        <main className="section">
+          <div className="container">
+            {pathname === '/home' && <Navigate to="/" replace={true} />}
```

Comment:
You can do it with Route
```suggestion
            <Route path="/home" element={<Navigate to="/" replace={true} />} />
```